src/services/openDataBot.service.js

```diff
@@ -0,0 +1,46 @@
+const config = require('../configs/index');
+const axios = require('axios');
+
+module.exports = {
+  async getCarBotIds({ carNumber }) {
+    const botURL = encodeURI(
+      `https://opendatabot.com/api/v2/transport?apiKey=${
```

Comment:
thank for fixing, 
Just one more  NIT: the url  `https://opendatabot.com/api/v2/transport` also could be saved  in some constant and be a bit more configurable